### ISSUE TITLE
False positive detection of apps with jetbrain-toolbox-app

### ISSUE THREAD

**Issue by 2743d2** [jysperm/homebrew-leftover-scanner#6]
96d68c6 as known issue

For app installed by jetbrain-toolbox-app, there could be mis-detection:

```
==> brew uninstall -f --zap intellij-idea
==> brew uninstall -f --zap jetbrains-gateway
==> brew uninstall -f --zap pycharm-ce-with-anaconda-plugin
==> brew uninstall -f --zap pycharm-ce
==> brew uninstall -f --zap pycharm-with-anaconda-plugin
==> brew uninstall -f --zap pycharm
==> brew uninstall -f --zap webstorm
```
i think the tool box made some magic,
I'm have these soft (idea...) installed but it was not in /Application
```
ls /Application
```

**Comment by 2743d2** [comment]
seems you are using package from upstream to get all installed apps.
> cask/caskroom

maybe it was the issue that brew or cast cant get apps not installed by brew or under `/Application`

**Comment by jysperm** [comment]
Yes, JetBrains Toolbox will installs .app to `/Library/Application Support/JetBrains/Toolbox` and can't be detected by this script. Currently, I have added it as a known issue in the README.
